util/create-payload.js

```diff
@@ -4,11 +4,10 @@ const extend = require('xtend')
 module.exports = createPayload
 
 
-function createPayload(data){
+function createPayload(data) {
   return extend({
     // defaults
     id: getRandomId(),
-    jsonrpc: '2.0',
```

Comment:
does the ws endpoint not accept this key?

Reply:
It does, but looking at the RPC over WebSocket spec I saw this was missing from requests and figured it better to be exact.